src/__tests__/fixtures/const-map-literal-type.thrift

```diff
@@ -22,6 +22,16 @@ const map<ShieldType, list<string>> THINGS = {
   ShieldType.U: ["uuuuuuu"],
 }
 
+const list<ShieldType> ITEMS = [
```

Comment:
Huh, syntax highlighting isn't happy with Flow syntax in .thrift files

Reply:
This is not even flow syntax tho, this is valid thrift

Reply:
Womp :(